src/core/qgsvectordataprovider.h

```diff
@@ -358,6 +358,14 @@ class CORE_EXPORT QgsVectorDataProvider : public QgsDataProvider, public QgsFeat
      */
     virtual QString defaultValueClause( int fieldIndex ) const;
 
+
+    /**
+     * \returns true if the given \a fieldIndex is a read-only field. This could be the case for
```

Comment:
```suggestion
     * Returns TRUE if the given \a fieldIndex is a read-only field. This could be the case for
```

then the various doc scripts correctly map it to "true" for the C++ docs or True for python docs